File: calculator.c
#include <stdio.h>

int main(){
    float num1, num2, result;
    int operator;

    printf("------ SIMPLE CALCULATOR ------\n\n");

    printf("Enter first number: ");
    scanf("%f", &num1);

    printf("\nChoose an operation:\n");
    printf("1. Addition\n2. Subtraction\n3. Multiplication\n4. Division\n");
    scanf("%d", &operator);  
    
    printf("\nEnter second number: ");
    scanf("%f", &num2);

    switch(operator){

        case 1:
            result = num1 + num2;
            printf("\nResult: %.2f\n", result);
            break;

        case 2:
            result = num1 - num2;
            printf("\nResult: %.2f\n", result);
            break;

        case 3:
            result = num1 * num2;
            printf("\nResult: %.2f\n", result);
            break;

        case 4:
            if(num2 != 0){
                result = num1 / num2;
                printf("\nResult: %.2f\n", result);
            } 
            else{
                printf("\nError! Division by zero is not possible.\n");
            }
            break;

        default:
            printf("\nInvalid operation! Please enter between 1-4\n");
    }

    return 0;
}

File: calculator.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Class.java to edit this template
 */
package com.mycompany.calculations;
import java.util.Scanner;

public class calculator {
    public static void main(String[] args){
        Scanner sc = new Scanner(System.in);
        
        System.out.println("-----SIMPLE CALCULATOR-----");
        
        System.out.print("Enter first number: ");
        int num1 = sc.nextInt();
        
        System.out.print("Enter second number: ");
        int num2 = sc.nextInt();
        
        System.out.println("Choose the operation!");
        System.out.println("1. Addition");
        System.out.println("2. Subtraction");
        System.out.println("3. Multiplication");
        System.out.println("4. Division");
        
        System.out.print("Enter you choice(1-4): ");
        int choice = sc.nextInt();
        
        switch(choice){
            case 1:
                int add = num1 + num2;
                System.out.println("Result: "+add);
            break;
               
            case 2:
                int sub = num1 - num2;
                System.out.println("Result: "+sub);
            break;
                
            case 3:
                int mul = num1*num2;
                System.out.println("Result: "+mul);
            break;    
                
            case 4:
                float div = (float)num1/num2;
                if(num2!=0){
                System.out.println("Result: "+div);
                }
                else{
                System.out.println("Error: Can not divide by zero!.");
                }
            break;    
                
            default:
                System.out.println("Invalid input! Can not perform operation.");
        }
    }
}
